bin/git-archive-file

```diff
@@ -26,7 +26,6 @@ fi
 
 # rename invalid chars for the file path
 FILENAME=${FILENAME//\//-}
-FILENAME=${FILENAME//\\/-}
```

Comment:
This line is different from the above

Reply:
Recovered

Reply:
Need more time for this case. The preset playground did not support the preset dirname.

Reply:
Alright, i take a wrong direction to figure the point out. There is only one way to pass the filter truly that `$BRANCH` has the slash or backslash. I will leave this in the issue.

Reply:
Done